i18n/translations.ts

```diff
@@ -39,4 +39,10 @@ export interface Translations {
   NotFound: { title: string; description: string; backHome: string };
   Unauthorised: { title: string; description: string };
   WorkInProgress: { title: string; description: string };
+  Departments: {
+    departmentTitles: string[];
+    readMore: string;
+    sectionTitles: string[];
+    subLinks: { key: string; link: string; title: string }[];
+  };
```

Comment:
I meant that this is not alphabetical with respect to the other elements like `WorkInProgress`. The inside stuff can be in any order.